scripts/process/compute_centroids.R

```diff
@@ -23,13 +23,14 @@ process.compute_centroids <- function(viz) {
   centroid_spdf <- sp::SpatialPointsDataFrame(centroids, spdat@data, match.ID=FALSE)
   
   # limit even more because centroids are a problem for our d3 projection?
-  centroid_spdf <- centroid_spdf[centroid_spdf$STATEFP %in% '04',]
+  # centroid_spdf <- centroid_spdf[centroid_spdf$STATEFP %in% '04',]
```

Comment:
I didn't want to completely delete this so that we could still remember it when we continue to iterate on the implementation working w/ d3 and the projection